feat(interview-prep): Converting and Transferring Rosetta problems (#38371)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/loop-over-multiple-arrays-simultaneously.md

---
id: 5e6dd15004c88cf00d2a78b3
title: Loop over multiple arrays simultaneously
challengeType: 5
---

## Description
<section id='description'>
Loop over multiple arrays and create a new array whose $i^{th}$ element is the concatenation of $i^{th}$  element of each of the given.
For this example, if you are given this array of arrays:
<code>[ ["a", "b", "c"], ["A", "B", "C"], [1, 2, 3] ]</code>
the output should be:
<code>["aA1","bB2","cC3"]</code>
</section>

## Instructions
<section id='instructions'>

> Write a function that takes an array of arrays as a parameter and returns an array of strings created as stated in the description.

Reviewer: ```suggestion
Write a function that takes an array of arrays as a parameter and returns an array of strings conforming to the description.
```
Maybe use an alternate to `conforming` if you think it might be hard for people.

Author: @thecodingaviator What do you think about this:
Write a function that takes an array of arrays as a parameter and returns an array of strings satisfying the given description.

Reviewer: That works for me
